### PR TITLE
Rexxar Mask

### PR DESCRIPTION
**Pull request by Chomski2**
## Changelog:
- Created a 3D model for **Rexxar's Mask**.
  - Made helmet usable by both genders.
  - Localized and added to in-game Barbershop.

<details><summary>Click to expand</summary>

image

</details>

## Tests:
- [ ] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [ ] The mod takes less than 5.5 GB in the Task Manager (Windows)

## How to test:
1. To view the model in-game, start by downgrading your **Crusader Kings III** to `1.4.4` and disabling both **Fate of Iberia** and **Royal Court** DLC.
2. Apply the helmet to any character using Barbershop.

### CODE REVIEW

**Review by ercarp — APPROVED** [review]
I'm personally happy with how the mask looks but definitely wait for a few more reviews before merging this into anything. In fact, I don't even know where it should be merged since it's on `1.4.1` but @zumbak04 probably does.

**Comment by Gaben1025** [comment]
> I'm personally happy with how the mask looks but definitely wait for a few more reviews before merging this into anything. In fact, I don't even know where it should be merged since it's on `1.4.1` but @zumbak04 probably does.

It’s probably going to be 1.4 bc that’s what the master is on. Whatever issues may come up will have to be fixed on the 1.5 adaptation

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Pull request merged**
